firebase-storage/CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Unreleased
+* [fixed] Fixed an issue where tests were depending on a deprecated feature of the test framework (#6927)
```

Comment:
No need to trigger a release for a test change, no?

Reply:
For simplicity, it's OK to do so.